src/Data/PreciseDateTime.purs

```diff
@@ -139,18 +141,18 @@ adjust pd (PreciseDateTime dt ns) = do
   pure (PreciseDateTime adjustedDateTime adjustedNs)
 
   where
-    zero = BigInt.fromInt 0
-    ten = BigInt.fromInt 10
-    tenPowNine = ten `pow` BigInt.fromInt 9
-    maxNano = tenPowNine - BigInt.fromInt 1
-    maxm = ten `pow` BigInt.fromInt 22
+    zero = Decimal.fromInt 0
+    ten = Decimal.fromInt 10
+    tenPowNine = ten `pow` Decimal.fromInt 9
+    maxNano = tenPowNine - Decimal.fromInt 1
+    maxm = ten `pow` Decimal.fromInt 22
 
-    -- | `Data.BigInt` only provides `toNumber`, which loses precision for
+    -- | `Data.Decimal` only provides `toNumber`, which loses precision for
     -- | numbers which are too large meaning that
-    -- | `Int.fromNumber <<< BigInt.toNumber` could produce a valid `Int` but
+    -- | `Int.fromNumber <<< Decimal.toNumber` could produce a valid `Int` but
     -- | would lose precision.
-    toInt :: BigInt -> Maybe Int
-    toInt = Int.fromString <<< BigInt.toString
+    toInt :: Decimal -> Maybe Int
+    toInt = Int.fromString <<< Decimal.toString
```

Comment:
Is this comment still accurate? Wondering if it's just the result of find and replace.

Reply:
It's a find and replace thing --- do we still need this function?

Reply:
It's still used in this module and there doesn't appear to be a `Decimal.toInt`.